pandas/core/series.py

```diff
@@ -3144,7 +3146,7 @@ def apply(self, func, convert_dtype=True, args=(), **kwds):
 
         >>> def square(x):
         ...     return x**2
```

Comment:
Can you fix the PEP-8 of all examples please? There are missing spaces around all the `**` , one `-` and one `+=`. I know it's from the original code, but let's get it fixed.